modules/storage-persistent-storage-pv.adoc

```diff
@@ -251,7 +251,7 @@ $ oc get pv <pv-claim>
 You can specify mount options while mounting a PV by using the attribute `mountOptions`.
 
 For example:
-
+ifndef::microshift[]
 .Mount options example
```

Comment:
Somehow in the OCP preview this is on the same line as "For example:" and is not displaying as the italic text heading, perhaps there should be a blank like above this one?

Reply:
Yes, reintroducing the blank line makes it work correctly again:

```
For example:

ifndef::microshift[]
.Mount options example
```